js/forum/src/components/DiscussionStickiedNotification.js

```diff
@@ -12,6 +12,6 @@ export default class DiscussionStickiedNotification extends Notification {
   }
 
   content() {
-    return app.translator.trans('flarum-sticky.forum.discussion_stickied_notification', {user: this.props.notification.sender()});
+    return '{username} stickied';
```

Comment:
Yeah ... I figured I'd replace the app.translator call in the meantime, since I was removing the key from the YAML file. :)